### PR TITLE
Sync'd the organization files and added transports for VT and NM and GU

### PR DESCRIPTION
**Pull request by MauriceReeves-usds**
This PR updates the organization files so they're matching across the different levels and updates VT, NM, and GU to have transport information in prod. This will require us to add their credential information into prod so we can test and verify the connections.

I also changed the name of TCP to match TPCA as it should be.

## Changes
- Changed the name of TCP to TPCA
- Added organizations for VT, NM, and GU to prod and included their transport info
- Updated the docs

## Checklist
- [ ] Tested locally?
- [ ] Ran `quickTest all`?
- [ ] Ran `./prime test` against local Docker ReportStream container?
- [ ] Downloaded a file from http://localhost:7071/api/download
- [ ] Updated the release notes? 
- [ ] Added tests?
- [ ] Did you check for sensitive data, and remove any? 
- [ ] Does logging contain sensitive data?  
- [ ] Are additional approvals needed for this change?
- [ ] Are there potential vulnerabilities or licensing issues with any new dependencies introduced?

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- Add the credential information for NM, GU, and VT into secrets storage on prod #712

### CODE REVIEW

File: prime-router/docs/schema_documentation/tpca-tpca-covid-19.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
-### Schema:         tcp/tcp-covid-19
-#### Description:   A COVID-19 schema for TCP working through A6
+### Schema:         tpca/tpca-covid-19
+#### Description:   A COVID-19 schema for TPCA working through A6
```
Comment (MauriceReeves-usds) [inline prime-router/docs/schema_documentation/tpca-tpca-covid-19.md:4]: I fully expect this to bomb the schema documentation check.

File: prime-router/settings/organizations-local.yml [hunk 2]
```diff
@@ -187,12 +187,12 @@
...
         topic: covid-19
-        jurisdictionalFilter: [ "matches(ordering_facility_state, FL)" ]
+        jurisdictionalFilter:
+          - matches(ordering_facility_state, FL)
         deidentify: false
         translation:
           type: HL7
           useBatchHeaders: true
-          schemaName: fl/fl-covid-19
           receivingApplicationName: FDOH-ELR
```
Comment (MauriceReeves-usds) [inline prime-router/settings/organizations-local.yml:196]: Removed schema for orgs receiving HL7

File: prime-router/settings/organizations-local.yml [hunk 3]
```diff
@@ -365,12 +362,11 @@
           - matches(ordering_facility_state, TX)
         translation:
           type: HL7
-          schemaName: tx/tx-covid-19
           useBatchHeaders: true
           receivingApplicationName: NEDSS
           receivingFacilityName: TX-ELR
         transport:
-          type: SFTP
+          type: SFTP_LEGACY
```
Comment (MauriceReeves-usds) [inline prime-router/settings/organizations-local.yml:369]: Updated TX everywhere to be SFTP_LEGACY

**Review by MauriceReeves-usds — COMMENTED** [review]
Added a few comments

**Comment by ronaldheft-usds** [comment]
Secrets are in place in prod now for this PR. #712

File: prime-router/settings/organizations-prod.yml [hunk 4]
```diff
@@ -280,7 +285,21 @@
         useBatchHeaders: true
       timing:
         operation: MERGE
-        numberPerDay: 12
+        numberPerDay: 1         # change to once a day for download site
+        initialTime: 01:15
```
Comment (jimduff-usds) [inline prime-router/settings/organizations-prod.yml:289]:
```suggestion
        initialTime: 09:15
```

0915 Eastern would be the same timing as AZ and Pima, and would clear any early morning submissions for a potential 9:30am deploy.

File: prime-router/settings/organizations-prod.yml [hunk 5]
```diff
@@ -280,7 +285,21 @@
...
+      jurisdictionalFilter:
+        - matches(patient_state, MT)
+      translation:
+        type: CUSTOM
+        format: CSV
+        schemaName: mt/mt-covid-19-csv
+      timing:
+        operation: MERGE
+        numberPerDay: 1
         initialTime: 01:15
```
Comment (jimduff-usds) [inline prime-router/settings/organizations-prod.yml:303]:
```suggestion
        initialTime: 09:15
```

File: prime-router/settings/organizations-prod.yml [hunk 6]
```diff
@@ -309,7 +328,61 @@
...
+        receivingFacilityName: GUDOH
+      transport:
+        type: SFTP
+        host: sftp-west.inductivehealth.com
+        port: 22
         filePath: ./
+      timing:
+        operation: MERGE
+        numberPerDay: 12 # every two hours
+        initialTime: 00:00
```
Comment (jimduff-usds) [inline prime-router/settings/organizations-prod.yml:356]:
```suggestion
        initialTime: 01:15
```
To match timing of other every-two-hour deliveries, again more likely to leave us free of data-in-transit

File: prime-router/settings/organizations-prod.yml [hunk 7]
```diff
@@ -309,7 +328,61 @@
...
+        receivingFacilityName: VDH
+        receivingFacilityOID: 2.16.840.1.114222.4.1.185
+      transport:
+        type: SFTP
+        host: gs-sftp.ahs.state.vt.us
+        port: 22
+        filePath: ./
+      timing:
+        operation: MERGE
+        numberPerDay: 1440 # Every minute
```
Comment (jimduff-usds) [inline prime-router/settings/organizations-prod.yml:383]:
```suggestion
        numberPerDay: 12
```

File: prime-router/settings/organizations-prod.yml [hunk 8]
```diff
@@ -309,7 +328,61 @@
...
+        initialTime: 00:00
```
Comment (jimduff-usds) [inline prime-router/settings/organizations-prod.yml:384]:
```suggestion
        initialTime: 01:15
```

**Review by jimduff-usds — APPROVED** [review]
I made some change suggestions to organizations-prod.yml  Can you take a look?    VT didn't look correct; the others I changed to match timings of other sends.

Is it intentional that TCP is not in -prod.yml yet?

**Pull request merged**